bootstraptest/test_ractor.rb

```diff
@@ -309,6 +309,16 @@ def test n
   end
 }
 
+# a ractor with closed outgoing port should terminate
+assert_equal 'ok', %q{
+  Ractor.new do
+    close_outgoing
+  end
+
+  true while Ractor.count == 1
```

Comment:
Did you mean `until` instead of `while`?

Reply:
😮